openff/toolkit/_tests/test_topology.py

```diff
@@ -606,7 +606,7 @@ def test_from_openmm_virtual_sites(self, opc):
 
         with pytest.raises(
             VirtualSitesUnsupportedError,
-            match=r"Atom <Atom 3 \(EP\) of chain 0 residue 0 \(UNK\)>.* a virtual site",
+            match=r"Atom <Atom 3 .*EP.*a virtual site",
```

Comment:
This change is to fix what appears to be a subtle behavior change in residue/chain numbering and some other changes in the text of this error

```
FAILED openff/toolkit/_tests/test_topology.py::TestTopology::test_from_openmm_virtual_sites - AssertionError: Regex pattern did not match.
 Regex: 'Atom <Atom 3 \\(EP\\) of chain 0 residue 0 \\(UNK\\)>.* a virtual site'
 Input: 'Atom <Atom 3 (EP) of chain 1 residue 1 (UNK)> has element None and is probably a virtual site. Virtual sites cannot be stored in `Topology` objects but can be stored in `Interchange` objects produced by `ForceField`s with virtual site parameters.'